### DISCUSSION TITLE
Welcome to Multi-channel-RCNN Discussions!

### DISCUSSION THREAD

**Discussion by OuYangg** [OuYangg/Multi-channel-RCNN#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by 15093985920** [comment]
想问一下大家整个项目运行大概时间/(ㄒoㄒ)/~~

**Reply by yejian95** [reply]
你好，我是这方面的新手。我想请教一下，我运行这个项目，并且在vscode交互式窗口可以输出损失图，但是后面一直卡住。您出现了这种情况了吗？

**Reply by 15093985920** [reply]
请问你是运行到哪里卡住了？我运行到309行这里MRCNN_model_list = Utils.compare_L(BA_1000_4,BA_1000_4_label,BA_1000_4_community,L_list)，后面报错Process finished with exit code -1073741819 (0xC0000005)

**Comment by yejian95** [comment]
您好，我可以加您个微信吗？我的微信号是JianGongLiYe95。我是这方面的新手，可以跟您探讨一下吗？谢谢&nbsp;


6166431
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2024年11月27日(星期三) 下午4:09
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [OuYangg/Multi-channel-RCNN] Welcome to Multi-channel-RCNN Discussions! (Discussion #1)














 
请问你是运行到哪里卡住了？我运行到309行这里MRCNN_model_list = Utils.compare_L(BA_1000_4,BA_1000_4_label,BA_1000_4_community,L_list)，后面报错Process finished with exit code -1073741819 (0xC0000005)

**Reply by 15093985920** [reply]
好的，已加
